python/ray/util/sgd/torch/examples/tune_example.py

```diff
@@ -56,6 +64,39 @@ def tune_example(operator_cls, num_workers=1, use_gpu=False):
 # __end_torch_tune_example__
 
 
+# __torch_tune_manual_lr_example__
+def tune_example_manual(operator_cls, num_workers=1, use_gpu=False):
+    def tune_func(trainer, iter):
```

Comment:
this should be `func_name(trainer, info: dict)` so that there are other parameters that can be provided later if needed.

Reply:
Yep makes sense, changed it to a dict. Though it only contains `iter` for now.